kuma/wiki/tests/__init__.py

```diff
@@ -149,6 +149,31 @@ def __iter__(self):
             .serialize(alphabetical_attributes=True))
 
 
+def create_document_editor_user():
+    """Creates a user empowered with document editing"""
+    perms = []
+    for action in ('add', 'change', 'delete', 'view', 'restore'):
+        perms.append(Permission.objects.get(codename='%s_document' % action))
+
+    group, group_created = Group.objects.get_or_create(name='editor')
+    if group_created:
+        group.permissions = perms
+        group.save()
+
+    User = get_user_model()
+    user, user_created = User.objects.get_or_create(
+        username='conantheeditor',
+        defaults=dict(email='user_%s@example.com',
```

Comment:
I mimicked `create_template_test_users` which had this as well. So, yes, it was intentional, but I have no idea why `create_template_test_users` does it.